### PR TITLE
Add "Art Requests" Guide

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

https://user-images.githubusercontent.com/99246719/226903145-179e15c8-18e9-4a93-91d4-64aa610bf632.mov

## Related Issue(s)

Art Requests issue template added in https://github.com/flowforge/.github/pull/16

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [-] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

File: src/handbook/design/art-requests.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+![Screenshot from GitHub to show the option for creating an Art Request](../images/screenshots/gh-artrequest-template.png "Screenshot from GitHub to show the option for creating an Art Request")
+
+#### Details to Include
+
+Please be sure to include as much detail as possible, but most importantly:
+
+- **Type**: What type of asset is this? Blog Tile? YouTube Thumbnail?
+- **Format**: Image (e.g. `.png`, `.jpg`), Video (e.g. `mp4`), Animation (e.g. `.gif` or Lottie Animtion)
+- **Deadline**: This is defined on the right-side of the issue under the "Art Requests" project, once the issue has been created.
```
Comment (iskerrett) [inline src/handbook/design/art-requests.md:36]: I can not see how I can add a deadline to this request https://github.com/flowforge/website/issues/545

Comment (joepavitt) [reply src/handbook/design/art-requests.md:36]: Screenshot 2023-03-23 at 09 09 34

It's on the right. I mention int he Handbook, the "Art Request" project gets added _after_ you create the issue, so it only becomes an available field then annoyingly.

Comment (ZJvandeWeg) [reply src/handbook/design/art-requests.md:36]: Thanks, I couldn't find it without the screenshot.

Comment (joepavitt) [reply src/handbook/design/art-requests.md:36]: (Screenshot is also included in the PR/Handbook)

**Pull request merged**